src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -288,6 +272,105 @@ void MemoryMngrRealloc::destroy(void *ptr) {
     dnnl::impl::free(ptr);
 }
 
+/////////////// StringMemory ///////////////
+
+StringMemory::StringMemory(const dnnl::engine& engine, const MemoryDescPtr& desc, const OvString* data) : m_engine(engine), m_mem_desc(desc) {
```

Comment:
We need to add precision check. Only string precision is allowed.

Reply:
Done